src/models/parties/parties.js

```diff
@@ -6,35 +6,43 @@ export const DEFAULT_PARTY_COUNT = 5;
 export const MAX_PARTY_COUNT = 10;
 
 export function makeParties() {
-  let names = Set([
-    'asparagus',
-    'corn',
-    'cabbage',
-    'turnip',
-    'radish',
-    'carrot',
-    'onion',
-    'spinach',
-    'mushroom',
-    'eggplant'
+  let potentialParties = Set([
+    {id: 'asparagus',
+    color: 'rgb(151, 190,17)'
+    },
```

Comment:
Small nit pic, but important to maintain style. Object attributes should be on a new line, tabbed one in than the `{`. So this should look like:
```js
{
  id: 'asparagus',
  color: 'rgb(151, 190, 17)'
}
```